hawkular-wildfly-agent/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
What is this file for? The agent shouldn't have to care about messaging-activemq existing, or actually anything in here that I can see.